### ISSUE TITLE
Log In Screen Pops up Frequently and takes Priority

### ISSUE THREAD

**Issue by BrendanJ** [keybase/keybase-issues#2510]
I'm a pretty big gamer, and on occasion I'll restart my computer during a gaming session, and attempt to hop right back into the game as soon as the computer is up. Sometimes when doing so, I'll disregard all my other applications that run on start up and request me to log into them, one of them being Keybase.

Generally if I'm not going to log in right away, I'll close the window prompting me for my keybase password. I've found though, no matter how many times I close it, it will reopen its self and take priority every so often (haven't paid attention to frequency, every 10-30 minutes). This is incredibly annoying when I'm playing a game competitively only to have the game minimized to see Keybase wants my password.

The first time it happened I thought maybe Keybase had just started up late, and I X'd out of the window. The second time it happened it was annoying but I thought surely it wouldn't continue. The 3rd and 4th time was pretty infuriating and only I could be blamed for not killing the entire Keybase process. However, I also feel it's something that shouldn't be happening at all. For my full screen game to be minimized numerous times for the Keybase login is pretty frustrating.

Windows 7 Ultimate x64
Motherboard name :  Z87 MPOWER MAX (MS-7815)
Manufacturer :  MSI
CPU:  Intel(R) Core(TM) i7-4770K CPU @ 4.0GHz

**Comment by malgorithms** [comment]
Agreed this is annoying.  We're going to change how this works soon, and it won't do this to you. (In the meantime, sorry!)
